apps/web/.gitignore

```diff
@@ -1,7 +1,9 @@
 .DS_Store
 node_modules
 /build
+/build-output
```

Comment:
I'd rename `build-output` to something less confusing, maybe `build-netlify`?

Reply:
seems oke